docs/docs/starlark-reference/plan.md

```diff
@@ -392,30 +392,27 @@ The `run_sh` instruction executes a one-time execution task. It runs the bash co
         # Sets the working dir in which:
         # the command will be run 
         # and files will be mounted at
-        # OPTIONAL (Default: /task)
+        # OPTIONAL (Default: "")
         workdir = "/task",
 
         # A mapping of path_on_task_where_contents_will_be_mounted -> files_artifact_id_to_mount
-        # The file path can be relative to workdir or absolute paths
         # For more information about file artifacts, see below.
```

Comment:
how does the workdir work with this now?

Reply:
removed workdir; so only absolute paths are supported.